File: NCC_API/Models/DanhMuc/InvestorModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Timensit_API.Models.DanhMuc
{
    public class InvestorModel
    {
        public long ID { get; set; } = 0;
        public string Name { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public string Address { get; set; }
        public string CitizenID { get; set; }
    }
}

File: Timensit_Backend/TimensitApp/src/app/views/pages/Timensit/danh-muc/investor/model/investor.model.ts
import { BaseModel } from '../../../../../../core/_base/crud';

export class investorModel extends BaseModel {
	ID: number;
	Name: string;
	Email: string;
	Phone: string;
	Address: string;
	CitizenID: string;
	clear() {
		this.ID = 0;
		this.Name = '';
		this.Email = '';
		this.Phone = '';
		this.Address = '';
		this.CitizenID = '';
	}
}
